src/org/opendatakit/briefcase/util/ExportToCsv.java

```diff
@@ -790,34 +793,34 @@ private boolean processInstance(File instanceDir) {
     EventBus.publish(new ExportProgressPercentageEvent((processedInstances * 100.0) / totalInstances, briefcaseLfd));
 
     // If we are encrypted, be sure the temporary directory
-    // that will hold the unencrypted files is created and empty.
+    // that will hold the unencrypted files is created.
     // If we aren't encrypted, the temporary directory
     // is the same as the instance directory.
 
     File unEncryptedDir;
     if (briefcaseLfd.isFileEncryptedForm()) {
-      // create or clean-up the temp directory that will hold the unencrypted
+      // create the temp directory that will hold the unencrypted
       // files. Do this in the outputDir so that the briefcase storage location
       // can be a read-only network mount. issue 676.
-      unEncryptedDir = new File(outputDir, ".temp");
-
-      if (unEncryptedDir.exists()) {
-        // silently delete it...
-        try {
-          FileUtils.deleteDirectory(unEncryptedDir);
-        } catch (IOException e) {
-          String msg = "Unable to delete stale temp directory: " + unEncryptedDir.getAbsolutePath();
-          log.warn(msg, e);
-          EventBus.publish(new ExportProgressEvent(msg, briefcaseLfd));
-          return false;
-        }
+      unEncryptedDir = null;
+      Path path = null;
+      try {
+        path = Files.createTempDirectory(Paths.get(outputDir.toURI()), ".temp");
+      } catch (IOException e) {
+        String msg = "Unable to get path of temp directory.";
+        log.error(msg, e);
+        EventBus.publish(new ExportProgressEvent(msg + " Cause : "
+                + e.toString(), briefcaseLfd));
+        return false;
       }
 
-      if (!unEncryptedDir.mkdirs()) {
+      unEncryptedDir = path.toFile();
+      if (!unEncryptedDir.exists()) {
```

Comment:
I think this whole `if` block is redundant and can be removed. I mean, if we get to this line, we can assume that the `Files.createTempDirectory` has worked.